packages/agreement-process/src/model/domain/agreement-validators.ts

```diff
@@ -430,10 +430,7 @@ export const validateSubmitOnDescriptor = async (
   eservice: EService,
   descriptorId: DescriptorId
 ): Promise<Descriptor> => {
-  const allowedStatus: DescriptorState[] = [
-    descriptorState.published,
-    descriptorState.suspended,
-  ];
+  const allowedStatus: DescriptorState[] = [descriptorState.published];
```

Comment:
While we're at it... :)
```suggestion
  const allowedState: DescriptorState[] = [descriptorState.published];
```